pymc/variational/opvi.py

```diff
@@ -93,14 +93,6 @@ class GroupError(VariationalInferenceError, TypeError):
     """Error related to VI groups"""
 
 
-class BatchedGroupError(GroupError):
```

Comment:
NotImplementedInference should also be removed

Reply:
There is a use of `NotImplementedInference`  bellow should we also remove that ?
https://github.com/pymc-devs/pymc/blob/4d2f3a8d5f99fc773bc0a1aad6d51e4f3f047698/pymc/variational/approximations.py#L274-L277

Reply:
should we remove `deterministic` parameter ?

Reply:
this deterministic parameter makes sense, I guess this was an aesara issue